proprietary/tokens/src/components/amsterdam/textbox.tokens.json

```diff
@@ -0,0 +1,11 @@
+{
+  "amsterdam": {
+    "textbox": {
+      "hover": {
+        "box-shadow": {
+          "value": "0 0 0 1px {amsterdam.color.primary-black}"
```

Comment:
In the current design the `box-shadow` is `inset` I think

Reply:
you're right, done